galsim/wfirst/wfirst_bandpass.py

```diff
@@ -87,6 +91,32 @@ def getBandpasses(AB_zeropoint=True, exptime=None, thin_err=1.e-4):
     ecliptic_lat = sky_data[0, :]
     ecliptic_lon = sky_data[1, :]
 
+    # Parse kwargs for truncation, thinning, etc., and check for nonsense.
+    truncate_kwargs = ['blue_limit', 'red_limit', 'relative_throughput']
+    thin_kwargs = ['rel_err', 'trim_zeros', 'preserve_range', 'fast_search']
+    tmp_truncate_dict = {}
+    tmp_thin_dict = {}
+    if 'default_thin_trunc' in kwargs.keys():
+        if len(kwargs) > 1:
+            raise ValueError("Cannot pass default_thin_trunc with other kwargs!")
+        if kwargs['default_thin_trunc']:
+            pass
+        else:
+            del kwargs['default_thin_trunc']
+    else:
+        if any(x in kwargs.keys() for x in truncate_kwargs):
+            for key in kwargs.keys():
+                if key in truncate_kwargs:
+                    tmp_truncate_dict[key] = kwargs[key]
+                    del kwargs[key]
```

Comment:
Minor simplification: These two lines can be combined as
```
tmp_truncate_dict[key] = kwargs.pop(key)
```
In fact, I think the whole block (including the if statement) can be just
```
tmp_truncate_dict = dict( ((key, kwargs.pop(key)) for key in kwargs if key in truncate_kwargs) )
```
But I'll let you decide if that dict comprehension is too confusing compared to the explicit loop, since efficiency isn't really a concern here.